lib/beaker-hostgenerator/generator.rb

```diff
@@ -29,9 +29,7 @@ def generate(layout, options)
 
       tokens.each do |token|
         if is_ostype_token?(token, bhg_version)
-          if nodeid[ostype] == 1 and ostype != nil
-            raise "Error: no nodes generated for #{ostype}"
-          end
+          raise "Error: no nodes generated for #{ostype}" if nodeid[ostype] == 1 and !ostype.nil?
```

Comment:
Can we disable the guard clause cop? I don't like it because of the long lines it generates. The old flow is much easier to understand IMHO.

Reply:
done